certified-connectors/Office 365 Users/apiDefinition.swagger.json

```diff
@@ -0,0 +1,282 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Office 365 Users",
+    "description": "Office 365 Users Connection provider lets you access user profiles in your organization using your Office 365 account. You can perform various actions such as get your profile, a user's profile, a user's manager or direct reports and also update a user profile."
+  },
+  "host": "graph.microsoft.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/codeless/v1.0/me": {
```

Comment:
Can we get rid of codeless in the path?  That will allow us to remove all policy templates. Do you know why and how is it used?

Reply:
This would be a breaking change. We can decide to add new set of all codeless operations with 'codeless' prefix removed.